src/frontend/apps/impress/src/i18n/translations.json

```diff
@@ -257,29 +225,29 @@
       "Untitled document": "Document sans titre",
       "Updated at": "Mise à jour le",
+      "Upload your docs to a Microsoft Word, Open Office or PDF document.": "Téléchargez vos documents dans un document Microsoft Word, Open Office ou PDF.",
       "Use as prompt": "Utiliser comme un prompt",
       "User {{email}} added to the document.": "L'utilisateur {{email}} a été ajouté au document.",
       "Validate": "Valider",
       "Version history": "Historique des versions",
       "Version restored successfully": "Version restaurée avec succès",
-      "Versions": "Versions",
       "Visibility": "Visibilité",
+      "Warning": "Attention",
       "We didn't find a mail matching, try to be more accurate": "Nous n'avons pas trouvé de correspondance par mail, essayez d'être plus précis",
       "We simply comply with the law, which states that certain audience measurement tools, properly configured to respect privacy, are exempt from prior authorization.": "Nous nous conformons simplement à la loi, qui stipule que certains outils de mesure d’audience, correctement configurés pour respecter la vie privée, sont exemptés de toute autorisation préalable.",
       "We try to respond within 2 working days.": "Nous essayons de répondre dans les 2 jours ouvrables.",
+      "Word / Open Office": "Word / Open Office",
       "You are the sole owner of this group, make another member the group owner before you can change your own role or be removed from your document.": "Vous êtes le seul propriétaire de ce groupe, faites d'un autre membre le propriétaire du groupe, avant de pouvoir modifier votre propre rôle ou vous supprimer du document.",
       "You can oppose the tracking of your browsing on this website.": "Vous pouvez vous opposer au suivi de votre navigation sur ce site.",
       "You can:": "Vous pouvez:",
       "You cannot update the role or remove other owner.": "Vous ne pouvez pas mettre à jour le rôle ou supprimer un autre propriétaire.",
-      "You don't have any document yet.": "Vous n'avez pas encore de document.",
       "Your current document will revert to this version.": "Votre document actuel va revenir à cette version.",
-      "Your role": "Votre rôle",
-      "Your role:": "Votre rôle:",
       "Your {{format}} was downloaded succesfully": "Votre {{format}} a été téléchargé avec succès",
       "accessibility-contact-defenseurdesdroits": "Contacter le délégué du<1>Défenseur des droits dans votre région</1>",
       "accessibility-dinum-services": "<strong>DINUM</strong> s'engage à rendre accessibles ses services numériques, conformément à l'article 47 de la loi n° 2005-102 du 11 février 2005.",
       "accessibility-form-defenseurdesdroits": "Écrire un message au<1>Défenseur des droits</1>",
       "accessibility-not-audit": "<strong>docs.numerique.gouv.fr</strong> n'est pas en conformité avec le RGAA 4.1. Le site n'a <strong>pas encore été audité.</strong>",
+      "doc title input": "entrée titre doc",
```

Comment:
It is not something dedicated for the tests ?

Reply:
Yes not wrong, what do you want to do? Should we remove the trad and put in hard?